modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/task/CreateClientExtensionConfigTask.java

```diff
@@ -205,17 +206,17 @@ public File getInputPluginPackagePropertiesFile() {
 		return GradleUtil.toFile(_project, "liferay-plugin-package.properties");
 	}
 
-	@InputFiles
+	@OutputFile
```

Comment:
This is an input file right?  the `${project}/LCP.json`

Reply:
There should be a corresponding input method called `getInputLcpJsonFile`.

Reply:
oh i see the `_lcpJsonFile` is set by the in the constructor via the _addTaskOutputFile()